src/components/icon/icon.njk

```diff
@@ -0,0 +1,27 @@
+<svg class="usa-icon usa-icon--size-3" focusable="false" aria-hidden="true" role="img">
```

Comment:
[dust] Do we need both aria-hidden and role here? Does aria-hidden override role and make it unnecessary?

Reply:
I left it in because mostly to ensure having both attributes in the tag. I think depending on where a user chooses to implement the icon, they can see it and decide to keep/remove depending on their use case. I had also made buttons where sometimes an icon is decorational since there's button text, but without text its not. That being said if someone just plucks it out of here and pastes it without evaluating that, then it's a good call that this might be unnecessary. I will definitely make this is correct as I build out components with these icons